### ISSUE TITLE
Report Errors - lp progress and exercises on course report

### ISSUE THREAD

**Issue by marshel** [chamilo/chamilo-lms#1877]
### Current behavior / Resultado actual / Résultat actuel
The report gives wrong info on progress and exercise as follows:

EXERCISE ISSUES:
The user has 2 exercises with 100% on both. The report show the promedio as 67%.
On the student frontend, it is show on the report the a non existent exercise.
Other students are ok, or at least looks like.
I'm trying to find others with same issue.

![2017-04-26 11_51_31-cob - treinadores iob - teoria e metodologia do treinamento no esporte_ metodos](https://cloud.githubusercontent.com/assets/1764993/25449666/d13c27de-2a92-11e7-8ebb-1df36fe1f655.png)
![2017-04-26 11_54_35-cob - treinadores iob - meu progresso](https://cloud.githubusercontent.com/assets/1764993/25449668/d1465916-2a92-11e7-907f-aebdb2f43c2b.png)
![2017-04-26 14_47_10-cob - treinadores iob - teoria e metodologia do treinamento no esporte_ metodos](https://cloud.githubusercontent.com/assets/1764993/25449667/d140e5bc-2a92-11e7-9003-f2cbaaeae670.png)

See that on other course, the exercise has two different results, but only one attempt.

![2017-04-26 15_09_48-cob - treinadores iob - testes medidas e avaliacao no esporte - detalhes da rot](https://cloud.githubusercontent.com/assets/1764993/25449836/6df68826-2a93-11e7-9265-04ca894d1f7b.png)

---------------------------
PROGRESS ISSUES:
On the same report on other course, the student has
![2017-04-26 15_08_49-cob - treinadores iob - fisiologia no esporte de alto rendimento](https://cloud.githubusercontent.com/assets/1764993/25449835/6df60310-2a93-11e7-88eb-a94e1a6fbf7e.png)
![2017-04-26 15_09_48-cob - treinadores iob - testes medidas e avaliacao no esporte - detalhes da rot](https://cloud.githubusercontent.com/assets/1764993/25449836/6df68826-2a93-11e7-9265-04ca894d1f7b.png)
See above that on the linepath (i)info,  the same report is showed correctly for the user.

![2017-04-26 15_09_10-cob - treinadores iob - testes medidas e avaliacao no esporte - acompanhamento](https://cloud.githubusercontent.com/assets/1764993/25449838/6dfa44c0-2a93-11e7-86e3-a3cbe4e394f6.png)
![2017-04-26 15_09_28-cob - treinadores iob - testes medidas e avaliacao no esporte - acompanhamento](https://cloud.githubusercontent.com/assets/1764993/25449837/6dfa2e68-2a93-11e7-9445-c65acf3d82f9.png)

### Expected behavior / Resultado esperado / Résultat attendu
Consistent and correct results on both student frontend and admin backend.

### Chamilo Version / Versión de Chamilo / Version de Chamilo
1.11.2

**Comment by AngelFQC** [comment]
Please try on 1.11.chamilo.org

**Comment by marshel** [comment]
@AngelFQC  Could you explain me the procedures to do so?
I tried to register but got an error as seen on the SS atached.
![2017-05-04 16_41_59-https___1 11 chamilo org_main_auth_inscription php](https://cloud.githubusercontent.com/assets/1764993/25721959/b48ff32a-30e8-11e7-9e2d-5abe93c43de6.png)

**Comment by AngelFQC** [comment]
Try again 😄 
Or enter with _admin_/_admin_

**Comment by marshel** [comment]
Ok, got registered, with some errors... 

How you suggest I try to reproduce?

**Comment by AngelFQC** [comment]
Create a course with users, exercises, learn paths, etc..

**Comment by ywarnier** [comment]
@marshel, if you can't reproduce it on 1.11.chamilo.org, we can't reproduce either, so I'll close the issue. I'm leaving you 3 days to update...

**Comment by marshel** [comment]
Sorry I could not reproduce. I did not had the time and there are for lot of variables to look at.

I will state some things I believe may be the cause so when someone reviews the code may look at it.
1st. - The linepaths are Generated on Articulate Then joined on Captivate for Scorm compliance.
2nd. - Then on Chamilo, is added to the linepath the 3 forums, 1 exercise.
3rd. - Most of the users can finish it without the glitch, with complete 100% status/progress. Some student on report are showing the progress errors.
4th. Lookint at the DB, the c_lp_item_view are correct, but the c_lp_view show error on the collumn progress.
5th. - There are multiple linepaths showing this behavior.

Things I ask myself:
- is the session time on php/apache a factor? (I.E. leaving the browser at a lp item for 90min for ex.)
- is the lp controller giving some users a error? (may posting or not posting on the forum, or yet on clicking or not on the home button)
- is the page that record the progress on the DB doing some wrong calculation or missing the record part?
- is the way that the Scorm is eddited the reason? but then, should not all the users have the same error?

I will keep looking and report when/if  I found something useful.

Thanks @ywarnier and @AngelFQC .
